sky/serve/schemas.py

```diff
@@ -0,0 +1,26 @@
+"""This file defines the schemas for the requests to the controller.
+"""
+
+from typing import List
+
+import pydantic
+
+from sky.serve import serve_utils
+
+
+class RequestAggregator(pydantic.BaseModel):
+    timestamps: List[float]
```

Comment:
timestamps is only used for `RequestTimestamp`, not all `RequestsAggregator`. We keep this for future extensibility. Can we have a list of schemas (length 1 since for now we only have one) and assert the request is in one of them?

https://github.com/skypilot-org/skypilot/blob/d5b6d89c83ea1ee7258f68314da4c6f8add83e04/sky/serve/serve_utils.py#L159-L163

Reply:
How about now?